### ISSUE TITLE
Failure to log in- Still

### ISSUE THREAD

**Issue by battlestrike2** [EDCD/EDDI#1069]
## EDDI version in which issue found
3.3.2 
Disconnected from VA at present
## Steps to reproduce

1. Trying to log in
2. Normal error of 'Unable to log in. This is normally an error with Frontier etc, etc'
## Expected
to Log in
## Investigation
Upgraded with a clean 3.2.2 version, no change

**Comment by Darkcyde13** [comment]
This is a known issue. FDev changed their login authentication and all third-party addons have been unable to log in for about a month.  The EDDI team are in the process of updating and hopefully this should be resolved in the very near future.

You should be aware that EDDI will work fine without being logged into the FDev API, just one or two things may be missing some data.  I'm still using it without problems, and I too am heading out on DW2.  I'm just on my way back to the bubble after a quick 6000LY round trip exploring, and have had no missing data from EDDI as far as I can tell.

**Comment by richardbuckle** [comment]
Thanks for the report. As Darkcyde13 said, this is in hand and will be resolved in a new release of EDDI this week.

**Comment by battlestrike2** [comment]
Thank you for this. No I didn’t realise EDDI would still work I shall try that today. I know I had clashes with EDDI and VA so I just stopped EDDI.

I put in my report because I thought version 3.2.2 fixed the error. Lack of understanding again on my part.

May catch you on DW2. I had to crash my ship yesterday because it had the ‘passenger bug’ and that was the only way I could get rid of it. 

Fly Safe Cmdr. and thanks again.
